### PR TITLE
Add includeRequestInErrorResponse to HttpMiddlewareOptions

### PR DESCRIPTION
**Pull request by ajimae**
### Summary
Add options to include or exclude original client request from error response

### Details
- [x] add an option - `includeRequestInErrorResponse` to httpMiddlewareOptions.
- [x] add test to validate functionality
- [x] add changeset to track changes

### Related Issues
[#240](https://github.com/commercetools/commercetools-sdk-typescript/issues/240)

### CODE REVIEW

File: packages/sdk-client/src/sdk-middleware-http/http.ts [hunk 1]
```diff
@@ -65,6 +65,7 @@ export default function createHttpMiddleware({
   credentialsMode,
   includeResponseHeaders,
   includeOriginalRequest,
+  excludeRequestInErrorResponse = false,
```
Comment (jenschude) [inline packages/sdk-client/src/sdk-middleware-http/http.ts:68]: I would stay consistent with the prefix include. So like `includeRequestInErrorResponse = true`

File: .changeset/small-numbers-peel.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools/sdk-client-v2': patch
+---
+
+add an option (`excludeRequestInErrorResponse`) to exclude original request from error responses.
```
Comment (jenschude) [inline .changeset/small-numbers-peel.md:5]: adjust the change set message please

**Review by jenschude — APPROVED** [review]
small note

File: .changeset/small-numbers-peel.md (hunk 2 above)
Comment (ajimae) [reply .changeset/small-numbers-peel.md:5]: Oh, sure I will do that 👍🏽

**Pull request merged**